File: Palindrome.js
// Write a function to check if a string is palindrome or not

let palindrome = (s) => {
    if (s.toLowerCase().split("").reverse().join("").replaceAll(" ", "") === s.toLowerCase().replaceAll(" ", "")) {
        return true;
    } else {
        return false;
    }
}

console.log(palindrome("Racecar"));

console.log(palindrome("Murder for a jar of red rum"));

File: Palindrome.py
#Write a function to check if a string is palindrome or not

def palindrome(s):
    if s.lower().replace(" ", "") == s.lower().replace(" ", "")[::-1]:
        return True
    else:
        return False

print(palindrome("Racecar"))

print(palindrome("Murder for a jar of red rum"))